person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
+
+  def can_use_services?
+    @age > 18 || @parent_permission
+  end
```

Comment:
- [ ] Please kindly take note that these two methods are marked as private, which means they can only be called from within the class itself and cannot be accessed directly from outside the class. This opposed the project requirement, indly ensure that `can_use_services?` is marked as public. 👍 

```suggestion
def can_use_services?
    @age > 18 || @parent_permission
  end
  
  private

  def of_age?
    @age > 18
  end  
```

Reply:
Thank you @OyePriscilla for the great insight and review 👍 I will make the required changes.